easybuild/easyblocks/o/openblas.py

```diff
@@ -26,7 +26,14 @@ def configure_step(self):
             'USE_OPENMP': '1',
             'USE_THREAD': '1',
         }
-        if LooseVersion(self.version) < LooseVersion('0.3.6') and get_cpu_architecture() == POWER:
+
+        if 'TARGET' in self.cfg['buildopts']:
```

Comment:
I would check for `TARGET=` to make it a little bit more strict (since you rely on `TARGET=` below basically)

Reply:
Done :+1: